.github/workflows/ci.yml

```
@@ -21,5 +77,48 @@ jobs:
           key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
           restore-keys: |
             ${{ runner.os }}-go
+      
       - name: Run Test Suite
-        run: go test -p=1 -race ./...
+        run: go test -p=1 -race ./... -coverpkg=./... -coverprofile=coverage.txt $1 | tee $OUTPUT_FILE
```

Perhaps out of scope, but why is this using `-p=1` to test one package at a time?
```suggestion
        run: go test -race ./... -coverpkg=./... -coverprofile=coverage.txt $1 | tee $OUTPUT_FILE
```

---

Thanks, removed `-p` flag